application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2437,6 +2474,12 @@ type TicketType implements Node {
   zones: [String!]
 }
 
+"A time window when a vehicle visit a stop, area or group of stops."
+type TimeWindow {
+  end: OffsetDateTime!
+  start: OffsetDateTime!
+}
```

Comment:
Are the start and end inclusive/exclusive?

Reply:
Start is easily answered: it is inclusive.

End is defined as "the time that the service ends" so technically the last pick up time would be 17:59:59.999 and not 18:00. It would look pretty odd in a travel plan though so I'm not sure that it's worth being very strict with the spec here.